src/mmw/js/src/draw/utils.js

```diff
@@ -159,6 +160,34 @@ function loadAsyncShpFilesFromZip(zipfile) {
         });
 }
 
+function isSelfIntersecting(shape) {
+    var selfIntersects = function(polygon) {
+        return turfKinks(polygon).features.length > 0;
+    },
+
```

Comment:
Extra whitespace here.

Reply:
Will remove before merging.

Thought we were supposed to add line breaks after functions because of [this comment](https://github.com/WikiWatershed/model-my-watershed/pull/1683#discussion_r99416015) from way back when.

Reply:
Sorry for the mixed messages on that,  I hadn't seen that comment before.  I left my comment only because I was momentarily confused about where `geom` was defined since I'm used to seeing the definitions not spaced.  Could have also just been the noise from the diff contributing to that.

Reply:
I think it looks better without the space in this case. Maybe when there are many functions declared it's better to separate them.